<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStudentDialog</class>
 <widget class="QDialog" name="addStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <widget class="QPushButton" name="addStudent_confirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 207, 8);
color: white;</string>
   </property>
   <property name="text">
    <string>Add Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAdd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 80, 64);
color: white</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>31</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Add Student</string>
   </property>
   <widget class="QLineEdit" name="addNameField">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addIDNumField">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year Level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="courseComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>203</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Course</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="genderBox">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>125</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Gender</string>
    </property>
    <widget class="QRadioButton" name="maleRadBut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>92</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="femaleRadBut">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>92</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="ageSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>165</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <number>15</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
